tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -162,20 +176,11 @@ def pre_build_jobspecs(self):
     return []
 
   def build_jobspec(self):
-    if self.platform == 'windows':
-      raise Exception("Not supported yet")
-    else:
-      if self.platform == 'linux':
-        environ = {}
-        if self.arch == 'x86':
-          environ['SETARCH_CMD'] = 'linux32'
-        return create_docker_jobspec(self.name,
```

Comment:
The way the build works currently is that the Jenkins job bypasses actually running task_runner.py -f artifact ruby linux and instead invokes build_artifact_ruby.sh directly. So this part of script actually isn't used at all.